python/chronos/src/bigdl/chronos/forecaster/__init__.py

```diff
@@ -13,7 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 import warnings
+# unset the KMP_INIT_AT_FORK
+# which will cause significant slow down in multiprocessing training
+import os
+os.unsetenv('KMP_INIT_AT_FORK')
```

Comment:
why do you need to unset it in both nano and chronos?

Reply:
We should avoid any `import torch` or with this env variable still exist.
In Chronos, nano trainer is imported really late (during `fit`), torch is imported before nano trainer's import, so I need to do it here.